_includes/header.html

```diff
@@ -41,8 +41,8 @@
                     <i class="fa fa-bars icon-custom"></i>
                 </button>
 
-                <a class="navbar-brand" href="index.html" title="Sterling Electrical Contractors of Seattle">
-                    <img src="images/sec-logo-white.png" alt="Sterling Electrical Contractors of Seattle">
+                <a class="navbar-brand" href="/" title="Sterling Electrical Contractors of Seattle">
+                    <img src="/images/sec-logo-white.png" alt="Sterling Electrical Contractors of Seattle">
```

Comment:
/images has a logo file and a background I stuck in it, but otherwise the same. I am not sure whey both. You can delete one.